letsencrypt-win-simple/Scripts/ImportExchange.ps1

```diff
@@ -1,56 +1,95 @@
 param(
     [Parameter(Position=0,Mandatory=$true)]
-    [string]$NewCertThumbprint,
+    [string]$NewCertThumbprint,         # should be {5} from letsencrypt.exe
     [Parameter(Position=1,Mandatory=$true)]
-    [string]$ExchangeServices
+    [string]$ExchangeServices,          # comma-separated list of exchange services to import for
+    [Parameter(Position=2,Mandatory=$false)]
+    [bool]$LeaveOldExchangeCerts = 1,   # 0 for false, 1 for true. If set to 1, will skip removal of old exchange certs
```

Comment:
Could we use `$true` and `$false` instead of 0 and 1?

Reply:
`$true` and `$false` are a pain to pass through from cmd. passing 0 and 1 are dead simple. 

Since it's a bool, powershell will interpret all those the same if you really want to.